docs/data-sources/consul_cluster.md

```diff
@@ -49,6 +49,7 @@ data "hcp_consul_cluster" "example" {
 - **region** (String) The region where the HCP Consul cluster is located.
 - **scale** (Number) The the number of Consul server nodes in the cluster.
 - **self_link** (String) A unique URL identifying the HCP Consul cluster.
+- **size** (String) The t-shirt size representation of each server VM that this Consul cluster is provisioned with. Valid option for development tier - x_small. Valid options for other tiers - small, medium, large.
```

Comment:
nit: not sure how much this matters but I noticed the style below with tier puts single quotes around the value options. Maybe we should do that here too for consistency?
```suggestion
- **size** (String) The t-shirt size representation of each server VM that this Consul cluster is provisioned with. Valid option for development tier - 'x_small'. Valid options for other tiers - 'small', 'medium', 'large'.
```

Reply:
Done